npm-debug.log

```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```

Comment:
This file should not be committed.  Again, please be cognizant of what you're committing.